api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -332,4 +354,23 @@ private User initializeUserIfNeeded() {
       return ResponseEntity.status(e.getCode()).build();
     }
   }
+
+  @Override
+  public ResponseEntity<Void> submitIdVerification(IdVerificationRequest request) {
+    Person person = blockscoreService.createPerson(
+      request.getFirstName(), request.getLastName(),
+      blockscoreService.createAddress(
+        request.getStreetLine1(), request.getStreetLine2(),
+        request.getCity(), request.getState(), request.getZip()
+      ),
+      request.getDob(),
+      request.getDocumentType(), request.getDocumentNumber()
+    );
+    User user = userProvider.get();
+    user.setBlockscoreId(person.getId());
+    userDao.save(user);
+    // TODO(dmohs): This should be 201 Created with no body, but the UI's swagger-generated code
+    // doesn't allow this. Fix.
+    return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
```

Comment:
Should we return ID verification status from this, or will it not have been set yet?

Reply:
I prefer having it show up in one and only one place, and /me seems appropriate.